spiceaidocs/docs/data-accelerators/duckdb.md

```diff
@@ -28,4 +28,14 @@ datasets:
       engine: duckdb
       params:
         duckdb_file: /my/chosen/location/duckdb.db
-```
\ No newline at end of file
+```
+
+## Limitations
+- Does not support schemas with [field types](https://duckdb.org/docs/sql/data_types): nested arrays/lists, structs or map fields. For example:
+  - Supported: 
+    - `SELECT [1, 2, 3];`
```

Comment:
@Jeadie - could you please double check this  on `eth.recent_transactions` - asling this as I think we saw this issue for `blob_versioned_hashes` column which is ARRAY type (list of strings).

Reply:
It appears that `blob_version_hashes` is a `List[Struct]]`

```
+-----------------------------+-----------------------------------------------------------------------------------------------------------------+-------------+
| column_name                 | data_type                                                                                                       | is_nullable |
+-----------------------------+-----------------------------------------------------------------------------------------------------------------+-------------+
| hash                        | Utf8                                                                                                            | NO          |
| nonce                       | Int64                                                                                                           | NO          |
| transaction_index           | Int64                                                                                                           | NO          |
| from_address                | Utf8                                                                                                            | NO          |
| to_address                  | Utf8                                                                                                            | NO          |
| value                       | Decimal128(38, 9)                                                                                               | NO          |
| gas                         | Int64                                                                                                           | NO          |
| gas_price                   | Int64                                                                                                           | NO          |
| input                       | Utf8                                                                                                            | NO          |
| receipt_cumulative_gas_used | Int64                                                                                                           | NO          |
| receipt_gas_used            | Int64                                                                                                           | NO          |
| receipt_contract_address    | Utf8                                                                                                            | YES         |
| receipt_root                | Utf8                                                                                                            | YES         |
| receipt_status              | Int64                                                                                                           | YES         |
| block_timestamp             | Timestamp(Microsecond, Some("UTC"))                                                                             | NO          |
| block_number                | Int64                                                                                                           | NO          |
| block_hash                  | Utf8                                                                                                            | NO          |
| max_fee_per_gas             | Int64                                                                                                           | NO          |
| max_priority_fee_per_gas    | Int64                                                                                                           | NO          |
| transaction_type            | Int64                                                                                                           | NO          |
| receipt_effective_gas_price | Int64                                                                                                           | NO          |
| max_fee_per_blob_gas        | Decimal128(38, 9)                                                                                               | YES         |
| blob_versioned_hashes       | List(Field { name: "item", data_type: Utf8, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }) | YES         |
+-----------------------------+-----------------------------------------------------------------------------------------------------------------+-------------+
```

Reply:
Found the issue. In spiceOSS we manually enforce non-nested types. We only need to check doubly nested types.

Reply:
Another issue. Utf8 is not a primitive type, not allowed either.

Reply:
Thank you Jack!

Reply:
Does it mean that `SELECT ['duck', 'goose', NULL, 'heron'];`  will be translated to list of Utf8 so won't work?